contrib/face_recognizer.go

```diff
@@ -11,7 +11,7 @@ import (
 	"gocv.io/x/gocv"
 )
 
-type FaceRecognizer interface {
+type IFaceRecognizer interface {
```

Comment:
I think a better name for the interface would be just `FaceRecognizer`

Reply:
Or if that is just too hard with the `type FaceRecognizer` already, name it `FaceRecognize` and the other interface `BasicFaceRecognize`